README.md

```diff
@@ -205,6 +205,18 @@ To avoid BIDI corruption that looks like "(Foo_(Bar", which happens when a strin
 
 The embedded context's directionality is determined by looking at the argument for `$1`, and then explicitly inserted into the Unicode text, ensuring correct rendering (because then the bidi algorithm "knows" the argument text is a separate context).
 
+## Wiki style links
+
+The message can have [mediawiki markup style links](https://en.wikipedia.org/wiki/Help:Wikitext#Links_and_URLs). By default this is disabled. To enables support for this, pass `wikilinks=true` option to `Banana` constructor. Example:
+
+```
+new Banana('es', { wikilinks:true } )
+```
+
+The original wiki links markup is elaborate, but here we only support simple syntax.
+
+* Internal links:  `[[pageTitle]]`  or `[[pageTitle|displayText]]`
```

Comment:
Does this give wiki/ prefix or how it works in this library? Maybe add an example output too to clarify.

Reply:
It is relative links `<a href="./Apple" title="Apple">Apple</a>`